README.rst

```diff
@@ -23,6 +23,8 @@ Roles
   Install of eups, and its required python packages
 :exim4:
   Configuration of exim4 as satellite host
+:fail2ban:
+  Install and configure fail2ban package (require option 'jail_enable')
```

Comment:
For clarifying, to have options, could you write an example in site.yml or somewhere? 
Or Are you planning to write something at some point?
I'm sorry.... I'm not so bright to your plan to make site.yml for operating ansible...

Reply:
Such document should go into docs/ ones, once PR #27 is finished, I think.